applications/minotari_app_grpc/proto/types.proto

```diff
@@ -119,7 +119,7 @@ message ConsensusConstants {
     uint64 median_timestamp_count = 9;
     uint64 emission_initial = 10;
     repeated uint64 emission_decay = 11;
-    uint64 emission_tail = 12;
+    uint64 emission_tail = 12 [deprecated=true];
```

Comment:
Nit:
I think we can just delete this, it should default to 0 if you ask for it in a dated proto file. 
But I am 99% nothing uses this field

Reply:
Let's nuke it in a separate PR so that it's easy to revert if you're wrong.